website/src/components/subscriptions/overview/SubscriptionEntry.tsx

```diff
@@ -26,8 +22,9 @@ export function SubscriptionEntry({ subscription }: { subscription: Subscription
                             <CardDescription
                                 title={subscription.name}
                                 subtitle={organismConfig[subscription.organism].label}
-                                icon={`size-6 ${getIcon()}`}
+                                icon={`size-8 ${getConditionIcon()}`}
                             />
```

Comment:
As discussed - a tooltip would be nice that explains the icon.

Reply:
![grafik](https://github.com/user-attachments/assets/36fffc31-4e73-445a-a63d-4ab2cf784074)